src/main/java/de/tum/in/www1/artemis/web/rest/iris/IrisSessionResource.java

```diff
@@ -108,4 +108,21 @@ public ResponseEntity<IrisSession> createSessionForProgrammingExercise(@PathVari
         var uriString = "/api/iris/sessions/" + session.getId();
         return ResponseEntity.created(new URI(uriString)).body(session);
     }
+
+    /**
+     * GET iris/sessions/{sessionId}/active: Retrieve if Iris is active for a session
+     * This checks if the used model is healthy.
+     *
+     * @return a status {@code 200 (Ok)} and with body true if Iris is active, false otherwise
+     */
+    @GetMapping("/sessions/{sessionId}/active")
+    @EnforceAtLeastStudent
+    public ResponseEntity<Boolean> isIrisActive(@PathVariable Long sessionId) {
```

Comment:
The doc comment is incomplete and I personally would include the information that this is just a temporary implementation.